### ISSUE TITLE
asusctl 4.x support

### ISSUE THREAD

**Issue by brawas** [Baldomo/asusctltray#5]
asusctltray wont start after recently update of asusctl

```
➜  ~ asusctltray
Traceback (most recent call last):
  File "/usr/local/bin/asusctltray", line 224, in <module>
    TrayIcon()
  File "/usr/local/bin/asusctltray", line 64, in __init__
    self.presets = self.config["power_profiles"].keys()
KeyError: 'power_profiles'

➜  ~ asusctl --version
App and daemon versions:
      asusctl v4.0.3
        asusd v4.0.3

Component crate versions:
    rog-anime v1.1.0
     rog-aura v1.1.0
     rog-dbus v4.0.2
 rog-profiles v1.1.2
rog-supported v4.0.0
  supergfxctl v2.0.0

Product family: ROG Zephyrus G14
Board name: GA401QM
```
have you any plans to update?

**Comment by Baldomo** [comment]
The current situation with `asusctl` is quite prone to changing (and I had a discussion with asusctl's developer on the Discord about the `4.x` release). [Many breaking changes](https://asus-linux.org/blog/updates-2021-07-16/#asus-wmi-add-support-for-platform-profile) were made both to the dbus interfaces and the configuration files (they're now in a TOML-like format instead of JSON), but I'm working on several fixes which I want to push a single commit, so for now `asusctltray` is completely broken. Said fixes are not trivial at all and will take some time and considerable code refactoring, but I'm positive I can manage. I've already fixed the dbus interfacing code and I'm currently tackling reading the configuration files as the python standard library does not support such a file format. And thank you for opening an issue, this will stay open and pinned as long as this project is in a broken state (or at least `asusctl` gets a stable 4.x API)

PS: my plans include adding a way to toggle turbo boost on demand, which was removed from `asusctl` in this latest release
